src/components/molecules/Visualizer/Widget/Timeline/index.tsx

```diff
@@ -0,0 +1,36 @@
+import TimelineUI from "@reearth/components/atoms/Timeline";
+import { ComponentProps as WidgetProps } from "@reearth/components/molecules/Visualizer/Widget";
+import { styled } from "@reearth/theme";
+
+import { useTimeline } from "./hooks";
+
+export type Props = WidgetProps;
+
+const Timeline = ({ widget, sceneProperty }: Props): JSX.Element | null => {
+  const { isOpened, currentTime, range, events } = useTimeline();
+
+  return (
+    <Widget extended={!!widget?.extended?.horizontally}>
+      <TimelineUI
+        isOpened={isOpened}
+        currentTime={currentTime}
+        range={range}
+        sceneProperty={sceneProperty}
+        {...events}
+      />
+    </Widget>
+  );
+};
+
+const Widget = styled.div<{
+  extended?: boolean;
+}>`
+  max-width: 100vw;
+  width: ${({ extended }) => (extended ? "100%" : "500px")};
```

Comment:
Is the 500px based on the design? I'm guessing you are copying this from Storytelling, which is good because I think they are very similar widgets, but please double check with @lavalse for a good base width.

Reply:
This was fixed by [style: fix width to 720px](https://github.com/reearth/reearth-web/pull/285/commits/8e45fb971b4f2f1dba37be89cbbc551455e48d54)